core/controller/src/main/resources/reference.conf

```diff
@@ -33,5 +33,6 @@ whisk {
   controller {
     protocol: http
     interface: "0.0.0.0"
+    readiness-fraction: 50%
```

Comment:
If you remove the `.toOption` (see other comment), this default I guess should be 100%?